Source/WebCore/platform/mediastream/RealtimeMediaSourceSettings.h

```diff
@@ -72,6 +72,24 @@ class RealtimeMediaSourceSettings {
     WEBCORE_EXPORT OptionSet<RealtimeMediaSourceSettings::Flag> difference(const RealtimeMediaSourceSettings&) const;
 
     explicit RealtimeMediaSourceSettings() = default;
+    RealtimeMediaSourceSettings(uint32_t width, uint32_t height, float aspectRatio, float frameRate, VideoFacingMode facingMode, double volume, uint32_t sampleRate, uint32_t sampleSize, bool echoCancellation, AtomString deviceId, AtomString groupId, AtomString label, DisplaySurfaceType displaySurface, bool logicalSurface, RealtimeMediaSourceSupportedConstraints supportedConstraints)
+        : m_width(WTFMove(width))
```

Comment:
Moving a width is probably not needed, let's move deviceId, groupId, label and supportedConstraints.